data/json/monsters.json

```diff
@@ -3076,6 +3076,52 @@
     "special_when_hit": [ "ZAPBACK", 75 ],
     "death_drops": "default_zombie_death_drops",
     "death_function": [ "NORMAL" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "ELECTRIC",
+      "CBM_SUBS",
+      "NO_BREATHE",
+      "REVIVES",
+      "BONES"
+    ],
+    "upgrades": { "half_life": 28, "into": "mon_zombie_nullfield" }
+  },
+  {
+    "id": "mon_zombie_nullfield",
+    "type": "MONSTER",
+    "name": "null zombie",
+    "description": "This once-human is visible only as a glowing white silhouette that you have to squint to see, cloaked in a crackling field of lightning that pulses like a beating heart.  It walks slowly and deliberately, the thunderstorm surrounding it eagerly jumping to anything conductive within its grasp.",
```

Comment:
This once-human?

Reply:
Once a human, now a zombie.

Reply:
I understand that part, just think you should clarify, or else it may sound like a grammatical error